### DISCUSSION TITLE
Log in After Installtion

### DISCUSSION THREAD

**Discussion by aishock** [open-webui/open-webui#5397] [General]
# Bug Report

## Installation Method

I installed it on Ubuntu using docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

## Problem

After installation following GitHub instructions, the app can be visisted on localhost. It requires email and password to log in, but I don't see any admin or default username and password. Anyone knows what to do next?

**Comment by bzikst** [comment]
You need to enable new sign ups and register a user. First one gets admin rights automatically
